src/main.cpp

```diff
@@ -121,28 +129,61 @@ auto main( int argc, char* argv[] ) -> int {
 					return 1;
 				}
 			} else {
-				Log_Warn( "Index file `{}` already exists, will be overwritten.", indexPath.string() );
+				Log_Warn( "Index file `{}` already exists, it will be overwritten.", indexPath.string() );
 			}
 			// clear contents
 			std::ofstream writer{ indexPath, std::ios::out | std::ios::trunc };
 		}
 
+		// if we're doing a steam build we want the default excludes, but not anything user-specified, because it
```

Comment:
This comment is now wrong, since the regexes are split between files and files in archives user-specified regexes are fine - this comment and the following code block can be removed